src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -1221,11 +1221,14 @@ class DevTask extends AbstractFeatureTask {
         return connection;
     }
 
-    static void runGradleTask(BuildLauncher buildLauncher, String ... tasks)  {
+    void runGradleTask(BuildLauncher buildLauncher, String ... tasks)  {
         buildLauncher
                 .setStandardOutput(System.out)
-                .setStandardError(System.err);
-        buildLauncher.forTasks(tasks);
+                .setStandardError(System.err)
+                .forTasks(tasks);
+        if (logger.isEnabled(LogLevel.DEBUG)) {
+            buildLauncher.withArguments("--debug");
```

Comment:
Consider `addArguments` since that would append the argument, whereas `withArguments` would override any previous args that are in the build launcher.

Reply:
Good catch! Thank you.